### DISCUSSION TITLE
Port is always in use.

### DISCUSSION THREAD

**Discussion by afbagwell** [vladmandic/sdnext#1813] [General]
During the middle of a run-of-the-mill upgrade on SD.Next, my computer completely froze. (I'm sure it was not the upgrade process that caused this. Windows. It just happens sometimes.)

Anyway, after a cold reboot of my PC, I try to start up SD.Next and I get an error:

`OSError: All ports from 7860 to 7959 are in use. Please close a port.`

However if I do:
netstat -ano | find "7860"

I don't get a PID identifying any other process using the port. 

So I tried a different port and :

`OSError: Port 17860 is in use.`

This repeats on any port I tried.

I thought maybe the computer freeze screwed up my git checkout. Here's what status tells me:

```
$ git status
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
  (commit or discard the untracked or modified content in submodules)
        modified:   extensions-builtin/sd-extension-system-info (untracked content)
        modified:   extensions-builtin/sd-webui-agent-scheduler (new commits)
        modified:   wiki (untracked content)

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        extensions-builtin/sd-webui-model-converter/

no changes added to commit (use "git add" and/or "git commit -a")
```

Git tells me I have the latest update, but I can't discard any of the changes listed as "modified" when I use the restore command.

Anyone got an idea of what else I can try?

**Comment by afbagwell** [comment]
FYI. I have tried rolling back to an earlier checkout then advancing to the most current again.

Even the early checkout gave me the port in use error.

**Reply by Seafan75** [reply]
Have a look at https://github.com/vladmandic/automatic/issues/1798.

**Reply by afbagwell** [reply]
Oh! I didn't even bother to look at issues since this all started for me right after a crash in the middle of a checkout. I figured this was all just a problem with my local checkout.

Thanks!
